packages/core/src/App/app.js

```diff
@@ -8,7 +8,18 @@ configure({ enforceActions: 'observed' });
 
 const setStorageEvents = root_store => {
     window.addEventListener('storage', evt => {
+        const active_loginid = root_store.client.loginid;
+
         switch (evt.key) {
+            case 'client.accounts':
+                if (
+                    document.hidden &&
+                    JSON.parse(evt.oldValue)[active_loginid]?.currency !==
+                        JSON.parse(evt.newValue)[active_loginid]?.currency
+                ) {
+                    window.location.reload();
```

Comment:
Is it possible to trigger the change without reloading the other tab?

Reply:
We need to connect the currency to websocket in whole application like `balance`. Do we have a api to do that?